docs/api/hooks.md

```diff
@@ -68,16 +67,71 @@ export const CounterComponent = () => {
 Using props to determine what to extract:
 
 ```jsx
-import React from 'react'
+import React, {useCallback} from 'react'
 import { useSelector } from 'react-redux'
 
-export const TodoListItem = props => (
-  const todo = useSelector(state => state.todos[props.id], [props.id])
+export const TodoListItem = ({id}) => (
+  const selector = useCallback(state => state.todos[id], [id])
+  const todo = useSelector(selector)
 
   return <div>{todo.text}</div>
 }
 ```
 
+With Reselect:
```

Comment:
I would give much more context about this and the following pattern, i.e. in which situations to use it